From linux-usb-devel-bounces@lists.sourceforge.net Sun Oct 29 16:38:54 2006
Message-ID: <89f574550610291638g1d3deb3fg9aeed58c49956de2@mail.gmail.com>
Date: Sun, 29 Oct 2006 16:38:31 -0800
From: Alex Sanks <alex@sanks.net>
To: linux-usb-devel@lists.sourceforge.net
MIME-Version: 1.0
Content-Disposition: inline
Subject: USB: ipaq: Add HTC Modem Support
Content-Type: text/plain; charset="us-ascii"

Adds support for HTC Smart Phones in modem mode (as opposed to sync
mode).  Loads and works with pppd on my T-Mobile SDA.

Signed-off-by: Alex Sanks <alex@sanks.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ipaq.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/serial/ipaq.c
+++ gregkh-2.6/drivers/usb/serial/ipaq.c
@@ -320,6 +320,7 @@ static struct usb_device_id ipaq_id_tabl
 	{ USB_DEVICE(0x0B05, 0x9200) }, /* ASUS USB Sync */
 	{ USB_DEVICE(0x0B05, 0x9202) }, /* ASUS USB Sync */
 	{ USB_DEVICE(0x0BB4, 0x00CE) }, /* HTC USB Sync */
+	{ USB_DEVICE(0x0BB4, 0x00CF) }, /* HTC USB Modem */
 	{ USB_DEVICE(0x0BB4, 0x0A01) }, /* PocketPC USB Sync */
 	{ USB_DEVICE(0x0BB4, 0x0A02) }, /* PocketPC USB Sync */
 	{ USB_DEVICE(0x0BB4, 0x0A03) }, /* PocketPC USB Sync */
